src/components/Reports/SubscriberHistory/SubscriberHistory.js

```diff
@@ -76,6 +77,20 @@ const SubscriberHistory = ({
           campaignsPerPage,
           currentPage,
         );
+
+        const tabsProperties = [
+          {
+            url: `/reports/subscriber-history?email=${subscriber.email}`,
+            descriptionId: 'subscriber_history.title',
+            active: true,
+          },
+          {
+            url: `/reports/subscriber-gdpr?email=${subscriber.email}`,
+            descriptionId: 'subscriber_gdpr.page_title',
+            active: false,
+          },
+        ];
```

Comment:
Is it possible to avoid code duplication here?

Reply:
But it's not duplicated. Each object has different values. I think that I don't understand the idea.